src/main/kotlin/de/cyface/collector/verticle/CollectorApiVerticle.kt

```diff
@@ -162,32 +168,119 @@ class CollectorApiVerticle(
      * information.
      * @return the created main `Router`
      */
-    private fun setupRoutes(storageService: DataStorageService, mongoClient: MongoClient): Router {
+    private fun setupRoutes(storageService: DataStorageService, mongoClient: MongoClient): Future<Router> {
         // Setup router
         val mainRouter = Router.router(vertx)
+        val promise = Promise.promise<Router>()
+
+        // API V3 - Sync (as to be executed before async V4 setup)
+        // (!) cleanup parameters, secrets (keys, salt) when removing `v3` (see Incentives API PR #2)
+        val apiRouterV3 = Router.router(vertx)
+        val mainRoutePathV3 = "/api/v3/*"
```

Comment:
hard-coded endpoint for `api/v3` as this will not change anymore as already deprecated.

I didn't want to spend another 1-2 hours just adding another paramter `endpoint.v3` which will be dropped soon and won't change.